qcodes/instrument_drivers/Keysight/keysight_b220x.py

```diff
@@ -0,0 +1,181 @@
+from qcodes import VisaInstrument
+from qcodes.utils.validators import MultiType, Ints, Enum, Lists
+
```

Comment:
Notebook:
For the ground port mode:
> Note: Using the port anyway will silently fail, that means it will not raise a python error. Only if you actively query the error queue of the instrument you will see the error.

Is it possible to implement such a check so that users see the error automagically?

Reply:
In principle yes, however I think none of the drivers automatically poll the status Byte of the instruments. So the behavior is not unique to this instrument. I just felt like this is an interesting case for the user to know.

In general i like the idea of polling instrument status, however since this behavior is common to all SCPI based instruments I’d say automatic status polling should be implemented in in one spot for all scpi instruments (some subclass of VisaInstrument maybe)

Reply:
@QCoDeS/core what's your opinion?

I, frankly, seen status polling in some particular drivers (but i don't remember where) in places where it is directly beneficial.

If you say that it's safer/better/whatever for the user to poll the status explicitly, then fine :)

Reply:
I now added a decorator around the function calls that automatically clears the status byte *before* the function call, and polls the status byte *after* the function call.

A warning is emitted to the user if the status byte is != 0.